packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -41,9 +41,9 @@ spec:
     pilot:
       enabled: {{ .Values.pilot.enabled }}
     policy:
-      enabled: {{ .Values.policy.enabled }}
+      enabled: {{ .Values.mixer.policy.enabled }}
     telemetry:
-      enabled: {{ .Values.telemetry.enabled }}
+      enabled: {{ .Values.mixer.telemetry.enabled }}
```

Comment:
If this is `spec.components.telemetry`, doesn't that correspond with values.telemetry.v1 or v2 ? I'm confused on both why this would be set via mixer and how v1 and v2 in values.yaml correspond to this. Same question for policy.

Reply:
Since `The functionality provided by Mixer is being moved into the Envoy proxies`, then the separate component installation responds to the deprecated mixer components. Envoy is managed via istiod so enable or disable occurs via the `.Values.telemetry.enabled` option.